src/core/java/com/enderio/core/common/menu/SyncedMenu.java

```diff
@@ -12,20 +12,20 @@
 import net.minecraft.world.inventory.AbstractContainerMenu;
 import net.minecraft.world.inventory.MenuType;
 import net.minecraft.world.inventory.Slot;
-
 import org.jetbrains.annotations.Nullable;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 public abstract class SyncedMenu<T extends EnderBlockEntity> extends AbstractContainerMenu {
 
-    @Nullable
-    private final T blockEntity;
+    @Nullable private final T blockEntity;
```

Comment:
restore original formatting
```suggestion
    @Nullable
    private final T blockEntity;
```